pkg/kubelet/pluginmanager/plugin_manager.go

```diff
@@ -109,7 +109,7 @@ var _ PluginManager = &pluginManager{}
 func (pm *pluginManager) Run(sourcesReady config.SourcesReady, stopCh <-chan struct{}) {
 	defer runtime.HandleCrash()
 
-	if err := pm.desiredStateOfWorldPopulator.Start(stopCh); err != nil {
+	if err := pm.desiredStateOfWorldPopulator.Start(stopCh, 2); err != nil {
```

Comment:
Nit: why 2? What does it mean?

A named constant or a comment can answer that.